src/lib/homestore_backend/hs_shard_manager.cpp

```diff
@@ -102,8 +102,44 @@ ShardManager::AsyncResult< ShardInfo > HSHomeObject::_create_shard(pg_id_t pg_ow
     });
 }
 
-ShardManager::Result< ShardInfo > HSHomeObject::_seal_shard(shard_id_t id) {
-    return folly::makeUnexpected(ShardError::UNKNOWN_SHARD);
+ShardManager::AsyncResult< ShardInfo > HSHomeObject::_seal_shard(ShardInfo const& info) {
```

Comment:
Once a shard is sealed, the chunk backing this shard should be returned to heapChunkSelector, right?
Can you point me who is doing this part?

Reply:
Yes this needs to be added.